examples/rust/Cargo.toml

```diff
@@ -4,8 +4,8 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-kuzu = {path="../../tools/rust_api"}
-arrow = {version="53", optional=true, default-features=false, features=[]}
+kuzu = { path = "../../tools/rust_api" }
+arrow = { version = "54", optional = true, default-features = false, features = [] }
```

Comment:
Does enabling the arrow feature mean the internal communication between the rust code and kuzu uses the arrow layout?

Reply:
No. It just adds support for exporting query results as arrow. arrow-rs is a large dependency, so I thought it made sense for it to be optional.

Reply:
Yes of course. I was trying to understand why it's in the example's deps as the code doesn't use it?

Reply:
According to git blame: https://github.com/kuzudb/kuzu/commit/9dedddc1eaa0a66f076414cb701784b88b8256ca. Basically just because the build cache is less efficient without it, or at least, it was at the time. For a while we were using a workspace so that the api tests and the example builds don't duplicate work; I think that might have been removed since I don't see evidence of it (I had been trying to avoid polluting the root of the repo so it was a little hacky).

Reply:
Found the workspace: https://github.com/kuzudb/kuzu/actions/runs/14398472589/workflow#L694

Reply:
I think it was also used in the main ci workflow at one point, but I guess that one never got removed.